.travis.yml

```diff
@@ -31,15 +31,21 @@ matrix:
       script: sbt \
         "++$TRAVIS_SCALA_VERSION clean" \
         "++$TRAVIS_SCALA_VERSION test" \
-        "++$TRAVIS_SCALA_VERSION docs/makeMicrosite" \
-        "++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues"
+        "++$TRAVIS_SCALA_VERSION docs/makeMicrosite"
+        # "++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues"
 
```

Comment:
What fails if we turn this check back on?

Reply:
Hi @johnynek, there are some changes around the `CanBuildFrom` that will trigger `IncompatibleSignatureProblem` and `IncompatibleMethTypeProblem`;

We also get `MissingClassProblem` because I had to relocate `com.twitter.algebird.MutableBackedMap`

Reply:
Actually, regarding the `com.twitter.algebird.MutableBackedMap` we can actually have it in the same namespace ... it will not be a prob.

Reply:
I think I'd rather take the 2.12-/2.13+ strategy used here: https://github.com/typelevel/paiges/pull/189 -- this code doesn't change a ton, so I think it shouldn't be too much of a pain.

I'm a big believer in compatibility.

Reply:
Yup I agree ... I like the ` 2.12-/2.13+`, less copying 😄